### PR TITLE
[UX] Centralizing docs around StimulusBundle

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

This goes with https://github.com/symfony/ux/pull/930

Basically, now that StimulusBundle exists, it is the "heart" of UX: installing it activates Stimulus in your app + integration with the UX packages. And so, I've shortened this document and moved most of the instructions into StimulusBundle instead.

Let me know your thoughts.

Thanks!

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I like the simplification. Thanks for that!

But we tend to avoid super short documents ... and this one is now 54-line long. Maybe it's too short? Would it make sense to combine it with some other article? Moreover, there are many concepts floating around front-end: Stimulus, UX, AssetMapper, Webpack Encore, etc. So, having many small documents could be confusing to readers. Thanks.

File: frontend/encore/simple-example.rst [hunk 1]
```diff
@@ -480,7 +478,8 @@ Encore supports many more features! For a full list of what you can do, see
 .. _`WebpackEncoreBundle Configuration`: https://github.com/symfony/webpack-encore-bundle#configuration
 .. _`Stimulus`: https://stimulus.hotwired.dev/
 .. _`Stimulus Documentation`: https://stimulus.hotwired.dev/handbook/introduction
-.. _`Symfony Stimulus Bridge`: https://github.com/symfony/stimulus-bridge
+.. _StimulusBundle & the UX System: https://symfony.com/bundles/StimulusBundle/current/index.html
+.. _all the Symfony UX packages: https://symfony.com/bundles/StimulusBundle/current/index.html#ux-packages
```
Comment (weaverryan) [inline frontend/encore/simple-example.rst:482]: Whenever StimulusBundle docs build next, this should be there. It's from https://github.com/symfony/ux/pull/930

**Comment by weaverryan** [comment]
Ready to go - I agree on removing the UX page entirely... I was also pondering that. It's done now - with a redirection to the StimulusBundle docs.

**Review by javiereguiluz — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Ryan.

**Pull request merged**